sample_max_time: 4.0
sample_min_time: 3.0
sample_time_step: 0.1
sample_lat_width: 3.75      # 3.75
sample_width_length: 0.25

weight_st_object: 10.0
weight_st_jerk: 5.0
weight_lt_offset: 50.0
weight_lt_acc: 5.0